src/prefect/logging/handlers.py

```diff
@@ -355,3 +359,42 @@ def close(self) -> None:
             worker.flush()
 
         return super().close()
+
+
+class PrefectConsoleHandler(logging.Handler):
+    def __init__(
+        self,
+        highlighter: Highlighter = PrefectConsoleHighlighter,
+        styles: Dict[str, str] = None,
+        level: Union[int, str] = logging.NOTSET,
+        console: Optional[Console] = None,
```

Comment:
Is this used for anything? We should at least note that highlighter/styles will be ignored if a custom console is passed.